MovieApp/Screens/Home/Views/ListCell/ListItemCell/ListItemCellCV.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ListItemCellCV.swift
+//  MovieApp
+//
+//  Created by MinhDev on 10/12/21.
+//
+
+import UIKit
+import Model
+
+class ListItemCellCV: UICollectionViewCell {
+	// MARK: - IbOutlets
+	@IBOutlet private weak var imageListItem: UIImageView!
+	// MARK: - Life cycles
+	override func awakeFromNib() {
+		super.awakeFromNib()
+		imageListItem.image = UIImage(named: "image 2")
```

Comment:
```suggestion
		imageListItem.image = UIImage(named: "icon_xxx")
```
@minh5896 Please define image named follow prefix *icon_*

Reply:
fix complete